crates/cli/src/config.rs

```diff
@@ -107,7 +107,7 @@ pub struct RawConfig {
 
 const DEFAULT_HOST: &str = "127.0.0.1:3000";
 const DEFAULT_PROTOCOL: &str = "http";
-const DEFAULT_HOST_NICKNAME: &str = "local";
+const DEFAULT_HOST_NICKNAME: &str = "testnet";
```

Comment:
We would need to update all 3 of these:
```
const DEFAULT_HOST: &str = "testnet.spacetimedb.com";
const DEFAULT_PROTOCOL: &str = "https";
const DEFAULT_HOST_NICKNAME: &str = "testnet";
```

Reply:
![image](https://github.com/clockworklabs/SpacetimeDB/assets/8840024/41984b5a-b031-42af-ad01-7ff9bf6f3283)
Thanks, got it